compass/Coordinator.java

```diff
@@ -227,6 +228,14 @@ private void setup() throws ArgumentException {
       throw new IllegalArgumentException("depth must be > 0");
     }
     log.info("Setting initial depth to: " + depth);
+
+    log.info("Validating Coordinator addresses.");
+    if (!Objects.equals(nodeInfoResponse.getCoordinatorAddress(), db.getRoot())) {
+      log.error("Coordinator Addresses do not match! {} vs. {}", nodeInfoResponse.getCoordinatorAddress(), db.getRoot());
```

Comment:
```suggestion
      log.warn("Coordinator Addresses do not match! {} vs. {}", nodeInfoResponse.getCoordinatorAddress(), db.getRoot());
```

May not be an error if config allows it